CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ---
 
+## Master
+
+* Add missing FontConvertible protocol conformance to default fonts template.
```

Comment:
Please add 2 spaces at the end of this line to make your name+PR info go in the next line in same paragraph :wink: